apps/platform-app/src/app/[locale]/(profile)/(components)/UserPhotoUploader/UserPhotoUploader.tsx

```diff
@@ -84,10 +84,13 @@ export const UserPhotoUploader = () => {
                 {t('changePicture')}{' '}
               </label>
             </Button>
-            <GithubUserPhotoUploader
-              setImage={setUserImage}
-              showError={setImageUploadError}
-            />
+            {/* TODO LinkedIn photo uploder */}
```

Comment:
**suggestion:** Or simpler, remove "Import from Github", and let's leave just "Upload my photo".